src/lib/Sympa/Request/Handler/add.pm

```diff
@@ -177,7 +177,7 @@ sub _report_user {
     my $sender = $request->{sender};
 
     # Notify the new list owner/editor
-    unless ($request->{quiet}) {
+    unless ($request->{quiet} or $request->{reception} eq 'nomail') {
```

Comment:
If `$request->{reception}` was not given, it'd be better to consider `default_owner_options.reception` or `default_editor_options.reception` parameter.

Reply:
Thanks for your advice @ikedas